core/Plugin.php

```diff
@@ -438,6 +438,10 @@ public function getMissingDependencies($piwikVersion = null)
             return array();
         }
 
+        if (!SettingsPiwik::isSystemValidationEnabled()) {
```

Comment:
Small tweak to not having to execute this on very request as we know on production there are no missing dependencies. Happy to change the name of the setting. We might want to add something like this eventually to more places as we know our system is correctly configured. Saves eg 1% per tracking request.